op-service/sources/engine_client.go

```diff
@@ -136,6 +137,9 @@ func (s *EngineAPIClient) NewPayload(ctx context.Context, payload *eth.Execution
 
 	e.Trace("Received payload execution result", "status", result.Status, "latestValidHash", result.LatestValidHash, "message", result.ValidationError)
 	if err != nil {
+		if strings.Contains(err.Error(), "forced head needed for startup") {
```

Comment:
There are two instances of `forced head needed for startup` in our code. It might be a good idea to turn them into a constant; otherwise, it will be quite troublesome to modify them later.

Reply:
Initially, the intention here was to verify the synchronization status in two steps, hence the different logic checks. The updated commit added retry logic and moved the FCU status update into the first `if` condition, allowing the status verification to be completed in a single step. There are no longer two separate logic checks.

Reply:
updated